mlflow/projects/backend/abstract_backend.py

```diff
@@ -0,0 +1,34 @@
+from abc import abstractproperty, abstractmethod, ABCMeta
+
+
+class AbstractBackend():
+    """
+    Abstract class for MLflow Project Execution Backend
+    A backend must implement the run method to launch a new job
+    """
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def run(self, active_run, uri, entry_point, parameters,
+            backend_config):
+        """
+        Submit an entrypoint. It must returns a SubmittedRun object to track the execution
+
+        :param active_run: Current MLflow run object. Everything will registred in this run.
+        :param uri: uri to the project (could be local or git uri).
+                    This is the parameter given to mlflow run command.
+        :param entry_point: name of the entry point to execute.
+        :param parameters: Dict of parameters to pass to the entry point
```

Comment:
```suggestion
        :param params: Dict of parameters to pass to the entry point
```

Just a small nit/suggestion on the argument name, for conciseness

Reply:
done